src/Raven.Server/RavenServer.cs

```diff
@@ -843,6 +876,7 @@ private bool CertificateCallback(object sender, X509Certificate cert, X509Chain
         public void RefreshClusterCertificateTimerCallback(object state)
         {
             RefreshClusterCertificate(state, RaftIdGenerator.NewId());
+            UpdateCertificateExpirationAlert();
```

Comment:
This will be a "sticky" alert. The user can dismiss it but it will pop up again in one hour. Also, if there are only 3 days left the alert becomes red.